libgalois/test/property-index.cpp

```diff
@@ -263,6 +311,31 @@ TestStringIndex(size_t num_nodes, size_t line_width) {
   it = nonuniform_index->UpperBound("aaak");
   KATANA_LOG_ASSERT(it != nonuniform_index->end());
   KATANA_LOG_ASSERT(typed_prop->GetView(*it) == "aaam");
+
+  return g;
+}
+
+std::unique_ptr<katana::PropertyGraph>
+ReloadGraph(std::unique_ptr<katana::PropertyGraph> g) {
+  auto uri_res = katana::Uri::MakeRand("/tmp/propertyfilegraph");
+  KATANA_LOG_ASSERT(uri_res);
+  std::string rdg_dir(uri_res.value().path());
+
+  auto write_result = g->Write(rdg_dir, "test command line");
+
+  if (!write_result) {
+    boost::filesystem::remove_all(rdg_dir);
+    KATANA_LOG_FATAL("writing result: {}", write_result.error());
+  }
+
+  katana::Result<std::unique_ptr<katana::PropertyGraph>> make_result =
+      katana::PropertyGraph::Make(rdg_dir, tsuba::RDGLoadOptions());
+  boost::filesystem::remove_all(rdg_dir);
```

Comment:
I'm not crazy about our dependence on `boost::filesystem`, but I've used `remove_all`.  So nice.